Azure/azure-application-gateway/ingest/parser.yml

```
@@ -0,0 +1,51 @@
+name: azure-application-gateway
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+
+  - name: parsed_date
+    external:
+      name: date.parse
+      properties:
+        # Maybe also it should be just `time` field in original message
+        input_field: "{{json_event.message.timeStamp}}"
+        output_field: date
+
+  - name: set_ecs_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          event.kind: "event"
+          event.category: ["network"]
+          event.type: ["access", "connection"]
+          "@timestamp": "{{parsed_date.date}}"
+          event.provider: "Azure"
+          event.action: "{{json_event.message.operationName}}"
```

`event.dataset` seems more suitable for this field:
```suggestion
          event.dataset: "{{json_event.message.operationName}}"
```